content/05.discussion.md

```diff
@@ -1,45 +1,53 @@
 ## Discussion
 
-We have introduced a novel computational approach that can map TWAS results into a representation learned from gene expression to infer cell type-specific features of complex phenotypes.
-Our key innovation is that we project association statistics through a representation and that representation is derived not strictly from measures of normal tissue but also cell types under a variety of stimuli and at various developmental stages.
-We found that this analysis using latent representations prioritized relevant associations, even when single gene-trait effects are not detected with standard methods.
-Projecting gene-trait and gene-drug associations into this common representation links drug-disease treatment pairs more accurately than the single-gene method we derived this strategy from; and the findings were more interpretable for potential mechanisms of action.
-Finally, we found that the analysis of associations through latent representations provided reasonable groupings of diseases and traits affected by the same transcriptional processes and highlighted disease-specific modules expressed in highly relevant tissues.
-
-
-In some cases, the features linked to phenotypes appear to be associated with specific cell types.
-Associations with such cell type marker genes may reveal cell types that are potentially causal for a phenotype with more precision.
+We have introduced a novel computational strategy that integrates statistical associations from TWAS with groups of genes (gene modules) that have similar expression patterns across the same cell types.
+Our key innovation is that we project gene-trait associations through a latent representation derived not strictly from measures of normal tissue but also cell types under a variety of stimuli and at various developmental stages.
+This improves interpretation by going beyond statistical associations to infer cell type-specific features of complex phenotypes.
+We found that our approach can identify disease-relevant cell types from summary statistics, and several disease-associated gene modules were replicated in eMERGE.
+Using a CRISPR screen to analyze lipid regulation, we found that our gene module-based approach can prioritize causal genes even when single gene associations are not detected.
+We interpret these findings with an omnigenic perspective of "core" and "peripheral" genes, suggesting that the approach can identify genes that directly affect the trait with no mediated regulation of other genes, and thus prioritize alternative and potentially more attractive therapeutic targets.
+Using our gene module perspective, we also integrated drug-induced transcriptional profiles, which allowed us to connect diseases, drugs, and cell types.
+We found that our method predicted FDA-approved drugs for various complex diseases more accurately than a single-gene approach.
```

Comment:
I am not sure how much it makes sense to hang your hat on this. I think it's less important that it's better (and not an argument we really want to have). It's just interesting that you can throw out a bunch of signal and it's not worse. Maybe not key to emphasize in the intro/discussion and instead something to deal with specifically in the results in a careful way? I could see this as something that a reviewer could really go after.

Reply:
Yes, I agree. I removed the last sentence.